### DISCUSSION TITLE
"Automatic game list updates"

### DISCUSSION THREAD

**Discussion by Darthagnon** [MikeMaximus/gbm#260] [General]
Uh-oh. 

I hope this will remain optional and not change GBM too much?

I don't currently use the Game List feature, as I install to custom directories with Steam Emulators (i.e. custom save paths) most of the time.

**Comment by MikeMaximus** [comment]
I make all new features optional and off by default when possible.  The automatic game list update feature is not being designed to mess with custom entries, more so a convenient way to automatically update configurations you already imported or get newly added ones.

This feature may not end up in the final release.

**Reply by Darthagnon** [reply]
Thank you!!!
